<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>PipelineViewBase</class>
  <widget class="QDockWidget" name="PipelineViewBase">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>776</width>
        <height>500</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
      <string notr="true"/>
    </property>
    <property name="windowTitle">
      <string>MLA Paddock Power | Plan Pipelines</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents">
      <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
        </sizepolicy>
      </property>
      <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
        <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
        <item>
          <layout class="QHBoxLayout" name="headerLayout">
            <item>
              <widget class="QToolButton" name="sketchPipelineButton">
                <property name="toolTip">
                  <string>Sketch a Pipeline â€¦</string>
                </property>
                <property name="text">
                  <string/>
                </property>
                <property name="icon">
                  <iconset>
                    <normaloff>../../../images/new-split-paddock.png</normaloff>../../../images/new-split-paddock.png</iconset>
                </property>
              </widget>
            </item>
            <item>
              <spacer name="toolbarSpacer">
                <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                  <size>
                    <width>40</width>
                    <height>20</height>
                  </size>
                </property>
              </spacer>
            </item>
          </layout>
        </item>
        <item>
          <layout class="QHBoxLayout" name="contentLayout" stretch="1,3,1">
            <property name="topMargin">
              <number>0</number>
            </property>
            <item>
              <widget class="QGroupBox" name="pipelineListGroupBox">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="title">
                  <string>Pipelines</string>
                </property>
                <layout class="QVBoxLayout" name="pipelinesVerticalLayout">
                  <item>
                    <widget class="PipelineLayerList" name="pipelineList"/>
                  </item>
                </layout>
              </widget>
            </item>
            <item>
              <widget class="QGroupBox" name="profileGroupBox">
                <property name="styleSheet">
                  <string notr="true">background-colour: rgb(255,255,255);</string>
                </property>
                <property name="title">
                  <string>Elevation Profile</string>
                </property>
                <layout class="QVBoxLayout" name="profileVerticalLayout">
                  <property name="spacing">
                    <number>0</number>
                  </property>
                  <property name="leftMargin">
                    <number>0</number>
                  </property>
                  <property name="topMargin">
                    <number>0</number>
                  </property>
                  <property name="rightMargin">
                    <number>0</number>
                  </property>
                  <property name="bottomMargin">
                    <number>0</number>
                  </property>
                  <item>
                    <widget class="ProfileDetails" name="profileDetails" native="true">
                      <property name="sizePolicy">
                        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                          <horstretch>0</horstretch>
                          <verstretch>0</verstretch>
                        </sizepolicy>
                      </property>
                      <property name="styleSheet">
                        <string notr="true"/>
                      </property>
                    </widget>
                  </item>
                </layout>
              </widget>
            </item>
          </layout>
        </item>
        <item>
          <spacer name="verticalSpacer">
            <property name="orientation">
              <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
              <size>
                <width>0</width>
                <height>0</height>
              </size>
            </property>
          </spacer>
        </item>
      </layout>
    </widget>
  </widget>
  <customwidgets>
    <customwidget>
      <class>ProfileDetails</class>
      <extends>QWidget</extends>
      <header>mlapp/src/widgets/profile_details/profile_details</header>
      <container>1</container>
    </customwidget>
    <customwidget>
      <class>PipelineLayerList</class>
      <extends>QListWidget</extends>
      <header>mlapp/src/widgets/feature_list/pipeline_layer_list</header>
    </customwidget>
  </customwidgets>
  <resources/>
  <connections/>
</ui>
